modules/manage.php

```diff
@@ -14,6 +14,24 @@
 
 add_action( 'jetpack_activate_module_manage', array( Jetpack::init(), 'toggle_module_on_wpcom' ) );
 add_action( 'jetpack_deactivate_module_manage', array( Jetpack::init(), 'toggle_module_on_wpcom' ) );
+add_action( 'customize_register', 'add_wpcom_to_allowed_redirect_hosts' );
+
+// Add wordpress.com to the safe redirect whitelist if the Manage module is enabled
+// so the customizer can `return` to wordpress.com if invoked from there.
+function add_wpcom_to_allowed_redirect_hosts( $domains ) {
+	if ( Jetpack::is_module_active( 'manage' ) ) {
+		add_filter( 'allowed_redirect_hosts', allow_wpcom_domain );
```

Comment:
This line is throwing a notice for me:
```
 Notice: Use of undefined constant allow_wpcom_domain - assumed 'allow_wpcom_domain' in /srv/www/wordpress-coderevue/wp-content/plugins/jetpack/modules/manage.php on line 23
```
Perhaps we need to either swap the declaration with the usage, or make it a string?